src/vs/workbench/browser/parts/titlebar/titleMenuControl.ts

```diff
@@ -126,8 +130,13 @@ export class TitleMenuControl {
 		};
 		updateTitleMenu();
 		this._disposables.add(titleMenu.onDidChange(updateTitleMenu));
-		this._disposables.add(quickInputService.onShow(() => this.element.classList.toggle('hide', true)));
-		this._disposables.add(quickInputService.onHide(() => this.element.classList.toggle('hide', false)));
+		this._disposables.add(quickInputService.onShow(this._setVisibility.bind(this, false)));
+		this._disposables.add(quickInputService.onHide(this._setVisibility.bind(this, true)));
```

Comment:
Readability wise this is confusing. Why do we do `this._setVisibility` to true when we're hiding. I see that you do `!show` but why not not do `this.elemet.classList.toggle('hide', show)` and have show be false when hiding and true when showing.